Automattic/wp-calypso
# Add Tracks events for HappOps

Adds HappOps Tracks events.

### To test

1. Create a new burner account, head to http://calypso.localhost:3000/me/account/close, and close your account. You should see the event `calypso_account_closed` fire.

2. Go to http://calypso.localhost:3000/help/contact and enter the word "Hi" into the message box. Submit the form — you should see the event `calypso_sibyl_support_without_suggestions_showing` fire.

3. Go to http://calypso.localhost:3000/help/contact and enter the phrase "Help with my CSS". Click any Sibyl suggestion that pops up and see the event `calypso_sibyl_first_question_click` fire. Then submit the form — see the event `calypso_sibyl_support_with_suggestions_showing` fire
